src/appmixer/wiz/core/UploadSecurityScan/UploadSecurityScan.js

```
@@ -0,0 +1,193 @@
+
+const uploadFile = async function(context, { url, fileContent }) {
+
+    const upload = await context.httpRequest({
+        method: 'PUT',
+        url,
+        data: fileContent, // stream upload is not implemented on the wiz side
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    });
+    await context.log({ stage: 'upload-finished', uploadData: upload.statusCode, fileContent});
```

ESLint

[error] A space is required before '}'.

(object-curly-spacing)